README.md

```diff
@@ -27,6 +27,7 @@ Buildkite emoji will be shown on both light or dark backgrounds, and at a small
 
 Emoji | Aliases
 ----- | -------
+<img src="img-buildkite-64/oxc.png" width="20" height="20" alt="oxc"/> | `:oxc:` | `:oxlint:` | `:oxc-rs:`
```

Comment:
Let's comma separate the aliases for consistency with the other entries here.

```suggestion
<img src="img-buildkite-64/oxc.png" width="20" height="20" alt="oxc"/> | `:oxc:`, `:oxlint:`, `:oxc-rs:`
```

Reply:
Thanks @joshuay03